tools/bluebanquise-playbook

```diff
@@ -48,6 +48,13 @@ elif [[ ! -f "${ANSIBLE_CONFIG}/playbooks/${playbook}.yml" ]]; then
     exit 1
 fi
 
+playbook_yml="${ANSIBLE_CONFIG}/playbooks/${playbook}.yml"
+if [[ $(grep -c 'hosts:.*{{.*target.*}}' ${playbook_yml}) -ne 0 ]]; then
+    echo -e "\e[31mWARNING: Usage of {{ target }} in hosts is deprecated."\
+         "Please update your playbook.\e[0m"
+    sleep 3
```

Comment:
Instead of deprecation, we can also drop the support of target:
```suggestion
    echo -e "\e[31mERROR: Usage of {{ target }} in hosts is deprecated."\
         "Please update your playbook.\e[0m"
    exit 1
```

Reply:
your script also override "-l" option which is the short option for --limit, probably not a good idea to override this

Reply:
Indeed, I will replace this with --list. Thanks.

Reply:
The script is fairly simple and use bash getopts, thus I cannot easily switch to `--list`. The --limit is already handled with `-n <nodeset>`, there is no need to use ansible-playbook's `-l` with the command.

Reply:
-n comply with bootset command, so ok for me.

Reply:
I would raise an error with target, since it is more than deprecated: it proves to be dangerous with playbooks.